monitoring/rid_qualifier/aircraft_state_replayer.py

```diff
@@ -0,0 +1,146 @@
+
+        all_test_payloads = [] # This holds the data that will be deilver
+        
+        test_reference_time = arrow.get(self.test_configuration.now)
+        test_start_time = arrow.get(self.test_configuration.test_start_time)
+        test_start_offset = test_start_time.shift(minutes =1)
+        test_start_offset_isoformat = test_start_offset.isoformat()
+
+        requested_flights = [] # This objects holds the modified aircraft state data, aircraft state data is read from disk and timestamps are modified to have recent ones. 
+        for state_data_index, current_disk_rid_state_data in enumerate(self.disk_rid_state_data): 
+            uss = usses[state_data_index]  
+
+
+            test_id = str(uuid.uuid4())
+            test_injection_path = '/tests/{test_id}'.format(test_id=test_id)
+            
+            disk_reference_time_raw = current_disk_rid_state_data['reference_time']
+            disk_reference_time = arrow.get(disk_reference_time_raw)
+
+            current_disk_rid_state_data['reference_time'] = test_reference_time.isoformat()
+            updated_timestamps_telemetry = []            
+            
+            timestamp_offset = test_start_offset - disk_reference_time
+                        
+            for telemetry_id, flight_telemetry in enumerate(current_disk_rid_state_data['flight_telemetry']['states']):
+
+                timestamp = (arrow.get(flight_telemetry['timestamp']) + timestamp_offset)
+
+                
+                flight_telemetry['timestamp'] = timestamp.isoformat()
+                updated_timestamps_telemetry.append(flight_telemetry)
+                        
+            flight_details =  current_disk_rid_state_data['flight_details']
+            operator_details = current_disk_rid_state_data['operator_details']
+            
+            operator_location = OperatorLocation(lat = operator_details['location']['lat'], lng = operator_details['location']['lng'])            
+            rid_flight_details = RIDFlightDetails(operator_id = operator_details['operator_id'], operator_location = operator_location, operation_description = flight_details['operation_description'] , serial_number = flight_details['serial_number'], registration_number = operator_details['registration_number'])
+            test_flight_details = TestFlightDetails(effective_after = test_start_offset_isoformat,details = [rid_flight_details])
+            test_flight = TestFlight(injection_id = str(uuid.uuid4()), telemetry = updated_timestamps_telemetry, details_responses = test_flight_details)
+            test_flight_deserialized = self.make_json_compatible(test_flight)
+            requested_flights.append(test_flight_deserialized)
+            test_payload = TestPayload(test_id = test_id, requested_flights = requested_flights)
+            test_payload_data_metadata = DeliverablePayloads(injection_path = test_injection_path, injection_payloads = test_payload)
```

Comment:
The type provided for `injection_payloads` does not match the field annotation

Reply:
Thank you for this, I have now fixed the typing, will resolve after testing